### PR TITLE
Feature/largest musltig

### PR DESCRIPTION
**Pull request by Dreyhh**
Introduces getLabelByLabelData method - use case: given an address show its label.
Introduces getContactSignersCount method - use case: get largest multisig members count.
Adds 'network' property to SmartVaults class - use case: filter signers by network.
Fix: use 'txid' when creating label for a completed proposal

### CODE REVIEW

File: src/SmartVaults.ts [hunk 1]
```diff
@@ -1882,4 +1886,54 @@ export class SmartVaults {
...
+   * @async
+   * @param {string} labelData - The label data (could be an address a trxid, etc).
+   * @returns {Promise<SmartVaultsTypes.PublishedLabel>} - 
+   * A promise that resolves to a PublishedLabel.
+   *
+   * @example
+   * const labels = await getLabelByLabelData("trxid");
+   */
+  async getLabelByLabelData(labelData: string): Promise<SmartVaultsTypes.PublishedLabel> {
+    await this.getLabels()
```
Comment (sebastianmontero) [inline src/SmartVaults.ts:1902]: Es necesario obtener todos los labels? no es posible generar el label identifier a partir del label data y solo obtener el label deseado?

File: src/SmartVaults.ts [hunk 2]
```diff
@@ -1882,4 +1886,54 @@ export class SmartVaults {
...
+   *
+   * @example
+   * const howManySigners = await getContactSignersCount("hexPubKey");
+   */
+  async getContactSignersCount(pubKey: string): Promise<number> {
+    const contactsFilter = filterBuilder()
+      .kinds(Kind.Contacts)
+      .authors(pubKey)
+      .toFilters()
+    const contactsEvents = await this.nostrClient.list(contactsFilter)
```
Comment (sebastianmontero) [inline src/SmartVaults.ts:1927]: strfry soporta el metodo count? Seria bueno poder utilizarlo en este caso

**Review by sebastianmontero — CHANGES_REQUESTED** [review]

File: src/SmartVaults.ts (hunk 2 above)
Comment (Dreyhh) [reply src/SmartVaults.ts:1927]: no soporta el NIP-45

File: src/SmartVaults.ts (hunk 1 above)
Comment (Dreyhh) [reply src/SmartVaults.ts]: done

**Pull request merged**